ee-installer/src/check_lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Should these helper scripts be checked into git? Or maybe a `tools` or something directory at root instead of `ee-installer/src/`? No need to serve these files in GitHub Pages

Reply:
Added ems-docs/tools, moved the scripts into there, and added code so the tools know where the `tools` directory is.